docs/usage-cli.rst

```diff
@@ -75,7 +75,7 @@ See below how sample configuration file could look like. The structure is the sa
    blueprint: api-description.apib
```

Comment:
⏫ Ouch, we're not ignoring `__pycache__` folders and they were added to your commit by accident. Please remove the files from the commit, I'll issue a PR to prevent this, at once.

Reply:
Removed 0006046c118db32c08e50288923a12500fc5984e